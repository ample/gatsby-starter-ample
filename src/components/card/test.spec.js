import React from "react"
import renderer from "react-test-renderer"

import { component as Card, fixtures } from "."

describe("Card", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Card {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
