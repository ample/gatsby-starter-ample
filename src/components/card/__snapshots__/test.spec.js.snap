// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="card text_alignment_left"
>
  <h3>
    Card Title
  </h3>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<p>Cras mattis consectetur purus sit amet fermentum. Curabitur blandit tempus porttitor. Maecenas sed diam eget risus varius blandit sit amet non magna.</p>",
      }
    }
  />
  <a
    className="button theme_arrow"
    href="#"
    rel="noopener"
    target="_blank"
  >
    Read more
    <svg
      viewBox="0 0 192 512"
    >
      <path
        d="M166.9 264.5l-117.8 116c-4.7 4.7-12.3 4.7-17 0l-7.1-7.1c-4.7-4.7-4.7-12.3 0-17L127.3 256 25.1 155.6c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0l117.8 116c4.6 4.7 4.6 12.3-.1 17z"
      />
    </svg>
  </a>
</div>
`;
